src/gui/guiFormSpecMenu.cpp

```diff
@@ -2799,19 +2765,18 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 
 	// Add tooltip
 	{
-		assert(!m_tooltip_element);
+		assert(!m_tooltip_element); //todo: do we still need this assert? (sadly irr_ptr does not allow to set a deleter and m_tooltip_element has to be ->remove()d)
```

Comment:
When `m_tooltip_element` is dropped (see `removeChildren`), it also has to be removed from its parent. Hence I'm unsure if this assert can be safely removed.